rust/opendp_tooling/src/codegen/mod.rs

```diff
@@ -8,7 +8,6 @@ use crate::{Argument, TypeRecipe};
 pub mod python;
 pub mod r;
 
-#[allow(dead_code)]
 pub fn write_bindings(base_dir: PathBuf, files: HashMap<PathBuf, String>) {
```

Comment:
This one _is_ dead code if the bindings feature is disabled. Could instead do:

```suggestion
#[cfg(feature = "bindings")]
pub fn write_bindings(base_dir: PathBuf, files: HashMap<PathBuf, String>) {
```

Reply:
I might be misunderstanding your request, but that doesn't work for me. This change:
```diff
 pub mod python;
 pub mod r;

+#[cfg(feature = "bindings")]
 pub fn write_bindings(base_dir: PathBuf, files: HashMap<PathBuf, String>) {
     for (file_path, file_contents) in files {
         File::create(base_dir.join(file_path))
```
produces two build errors like this:
```
$ cargo build --features untrusted,bindings
...
error[E0425]: cannot find function `write_bindings` in module `codegen`
  --> build/derive.rs:39:22
   |
39 |             codegen::write_bindings(base_dir, codegen::python::generate_bindings(&_modules));
   |                      ^^^^^^^^^^^^^^ not found in `codegen`
   |
note: found an item that was configured out
  --> /Users/chuckmccallum/github/opendp/opendp/rust/opendp_tooling/src/codegen/mod.rs:12:8
   |
12 | pub fn write_bindings(base_dir: PathBuf, files: HashMap<PathBuf, String>) {
   |        ^^^^^^^^^^^^^^
   = note: the item is gated behind the `bindings` feature
```

My first guess was that downstream usages would also need to be gated, but adding `#[cfg(feature = "bindings")]` in `derive.rs` doesn't change anything. Maybe also need a change in `build/main.rs`?